packages/config/css/theme.css

```
@@ -0,0 +1,565 @@
+   * This will generate colors only for border utilities
+   * i.e. bg-muted and they'll only have border-color: var(--border-muted) rule
+   */
+  --border-color-default: hsl(var(--border-default));
+  --border-color-overlay: hsl(var(--border-overlay));
+  --border-color-muted: hsl(var(--border-muted));
+  --border-color-control: hsl(var(--border-control));
+  --border-color-strong: hsl(var(--border-strong));
+  --border-color-stronger: hsl(var(--border-stronger));
+  --border-color-secondary: hsl(var(--border-secondary));
+  --border-color-alternative: hsl(var(--border-alternative));
+  --border-color-strong: hsl(var(--border-strong));
```

Stylelint (17.11.0)

[error] Duplicate custom property "--border-color-strong" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)